<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ICHCOtherPage</class>
 <widget class="QWidget" name="ICHCOtherPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Other</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="clearButton">
     <property name="styleSheet">
      <string notr="true">::indicator {
     width: 32px;
     height: 32px;
 }

::indicator:unchecked 
{
     image: url(:/resource/check_unchecked.png);
 }
::indicator:checked 
{
     image: url(:/resource/check_checked.png);
 }

::indicator:checked:disabled
{
 image: url(:/resource/check_checked_disable.png);
}

::indicator:unchecked:disabled
{
 image: url(:/resource/check_unchecked_disable.png);
}</string>
     </property>
     <property name="text">
      <string>Product Clear</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="shakeGroup">
     <property name="styleSheet">
      <string notr="true">#shakeGroup::indicator {
     width: 32px;
     height: 32px;
 }

#shakeGroup::indicator:unchecked 
{
     image: url(:/resource/check_unchecked.png);
 }
#shakeGroup::indicator:checked 
{
     image: url(:/resource/check_checked.png);
 }

#shakeGroup::indicator:checked:disabled
{
 image: url(:/resource/check_checked_disable.png);
}

#shakeGroup::indicator:unchecked:disabled
{
 image: url(:/resource/check_unchecked_disable.png);
}</string>
     </property>
     <property name="title">
      <string>Shake</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Axis</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>speed</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Width</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QCheckBox" name="rPbox">
        <property name="styleSheet">
         <string notr="true">::indicator {
     width: 32px;
     height: 32px;
 }

::indicator:unchecked 
{
     image: url(:/resource/check_unchecked.png);
 }
::indicator:checked 
{
     image: url(:/resource/check_checked.png);
 }

::indicator:checked:disabled
{
 image: url(:/resource/check_checked_disable.png);
}

::indicator:unchecked:disabled
{
 image: url(:/resource/check_unchecked_disable.png);
}</string>
        </property>
        <property name="text">
         <string>RP</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>ST</string>
        </property>
       </widget>
      </item>
      <item row="2" column="6">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Times</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>mm</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Delay</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="4">
       <widget class="QComboBox" name="axisSel">
        <item>
         <property name="text">
          <string>X1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Y1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Z</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>X2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Y2</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1" colspan="4">
       <widget class="ICLineEditWithVirtualNumericKeypad" name="widthEdit">
        <property name="text">
         <string>0.00</string>
        </property>
       </widget>
      </item>
      <item row="3" column="6">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>%</string>
        </property>
       </widget>
      </item>
      <item row="4" column="6">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>s</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="4">
       <widget class="ICLineEditWithVirtualNumericKeypad" name="stEdit">
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="4">
       <widget class="ICLineEditWithVirtualNumericKeypad" name="delayEdit">
        <property name="text">
         <string>0.00</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="4">
       <widget class="ICLineEditWithVirtualNumericKeypad" name="speedEdit">
        <property name="text">
         <string>80</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ICLineEditWithVirtualNumericKeypad</class>
   <extends>QLineEdit</extends>
   <header>iclineeditwithvirtualnumerickeypad.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
